app/driver.go

```diff
@@ -206,15 +205,15 @@ func deployCSIDriver(kubeClient *clientset.Clientset, c *cli.Context, managerIma
 
 	defer func() {
 		var wg sync.WaitGroup
-		notify := func(kubeClient *clientset.Clientset, f func(*clientset.Clientset)) {
-			f(kubeClient)
-			wg.Done()
-		}
-
-		wg.Add(3)
-		go notify(kubeClient, attacherDeployment.Cleanup)
-		go notify(kubeClient, provisionerDeployment.Cleanup)
-		go notify(kubeClient, pluginDeployment.Cleanup)
+		util.RunAsync(&wg, func() {
```

Comment:
I know this is now an off-topic refactor, but an older revision of the code (that created a webservice and resources) relied on the same logic, so I genericized the function. I can split this into a separate commit (or remove) if need be